global-files/CODE_OF_CONDUCT.md

```diff
@@ -6,7 +6,7 @@ Exercism is a platform centered around empathetic conversation. We have a low to
 
 ## Seen or experienced something uncomfortable?
 
-If you see or experience abuse, harassment, discrimination, or feel unsafe or upset, please email abuse@exercism.org. We will take your report seriously.
+If you see or experience abuse, harassment, discrimination, or feel unsafe or upset, please email abuse@exercism.org and include \[CoC\] in the subject line. We will deal with all CoC issues as a priority.
```

Comment:
I'd prefer `[abuse@exercism.org](mailto:abuse@exercism.org?subject=COC+violation)` or something along those lines.

Reply:
Linking it is a good call! As for that specific suggestion, I'll wait for confirmation before doing that since that would require updating in other places as well then.

Reply:
Subject should stay as `[COC]` as there's stuff that filters those the other end, but agree with adding a link if we can 🙂

Reply:
(I was too lazy to look up the encoding for those braces :p)